cmd/subnetcmd/addValidator.go

```diff
@@ -53,6 +54,7 @@ This command currently only works on subnets deployed to the Fuji testnet.`,
 	cmd.Flags().Uint64Var(&weight, "weight", 0, "set the staking weight of the validator to add")
 	cmd.Flags().StringVar(&startTimeStr, "start-time", "", "UTC start time when this validator starts validating, in 'YYYY-MM-DD HH:MM:SS' format")
 	cmd.Flags().DurationVar(&duration, "staking-period", 0, "how long this validator will be staking")
+	cmd.Flags().BoolVarP(&useTestnet, "fuji", "f", false, "add validator to fuji")
```

Comment:
We should also have a `--testnet` flag that also sets the `useTestnet` variable

Reply:
done with merge